Assets/Scripts/RobotProgramming/LogEntry.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using UnityEngine;
+
+namespace Cosmobot
+{
+    public struct LogEntry
+    {
+        public long timestamp;
+        public LogLevel level;
+        public string message;
+
+        public LogEntry(long timestamp, LogLevel level, string message)
+        {
+            this.timestamp = timestamp;
+            this.level = level;
+            this.message = message;
+        }
+
+        public string GetIsoTime()
+        {
+            return DateTimeOffset.FromUnixTimeSeconds(timestamp).ToString("HH:mm:ss");
```

Comment:
```suggestion
            return DateTimeOffset.FromUnixTimeSeconds(timestamp).ToLocalTime().ToString("HH:mm:ss");
```
This returns UTC+0 time instead of actual timezone time